README.md

```diff
@@ -1,15 +1,60 @@
+Overview
+--------
+[![Build CI](https://github.com/tlaplus/tlaplus/workflows/CI/badge.svg?branch=master)](https://github.com/tlaplus/tlaplus/actions?query=workflow%3ACI)
 
-For more information, visit http://lamport.azurewebsites.net/tla/tla.html
+This repository hosts the core TLA⁺ CLI tools and the Toolbox IDE.
+Its development is managed by the [TLA⁺ Foundation](https://foundation.tlapl.us/).
+See http://tlapl.us for more information about TLA⁺ itself.
+For the TLA⁺ proof manager, see https://github.com/tlaplus/tlapm.
+
+This repository follows a rolling release model where every commit to the master branch is built & uploaded to the [1.8.0 Clarke pre-release](https://github.com/tlaplus/tlaplus/releases/tag/v1.8.0).
+Past versioned releases can be found on the [Releases](https://github.com/tlaplus/tlaplus/releases) page.
+
+The CLI tools and Toolbox IDE are both written in Java.
+The Toolbox is based on the [Eclipse Platform](https://github.com/eclipse-platform/eclipse.platform).
+The CLI tool source code is primarily located in the [tlatools/org.lamport.tlatools](./tlatools/org.lamport.tlatools) directory.
+The Toolbox IDE source code is contained within the [toolbox](./toolbox) directory.
+
+Build & Test CLI Tools
+----------------------
+Install the following dependencies to your path:
+ * [OpenJDK](https://openjdk.org/) version 11+
+ * [Apache Ant](https://ant.apache.org/) version 1.9.8+
+
+Clone this repo & open a shell in the `tlatools/org.lamport.tlatools` directory, then:
+ 1. To build, run `ant -f customBuild.xml compile dist`
+ 2. To test, run `ant -f customBuild.xml compile-test test`
+
+The unit tests should run for 4-10 minutes depending on how fast your computer is, and succeed.
+The compiled `tla2tools.jar` will be output to `tlatools/org.lamport.tlatools/dist/tla2tools.jar`, and can be used the same as any `tla2tools.jar` you download from the releases.
+You can try it on an example model with:
+
+```sh
+java -cp dist/tla2tools.jar tlc2.TLC test-model/pcal/Bakery.tla
+```
+
+Build & Test Toolbox IDE
+------------------------
+*Note: these instructions are likely to fail and must be fixed; see [this issue](https://github.com/tlaplus/tlaplus/issues/851).*
+
+Install the following dependencies to your path:
+ * [OpenJDK](https://openjdk.org/) version 11+
+ * [Apache Ant](https://ant.apache.org/) version 1.9.8+
+ * [Apache Maven](https://maven.apache.org/) version 3.9.7+
+
+Clone this repo & open a shell in its root, then:
+ 1. To build, run `mvn install -Dmaven.test.skip=true`
+ 1. To test, run `mvn verify`
+
+The Toolbox distributables will be output in the `toolbox/org.lamport.tla.toolbox.product.product/target/products` directory.
 
 Contributing
 ------------
-Please read our [contribution guide](https://github.com/tlaplus/tlaplus/blob/master/CONTRIBUTING.md) before you start working on your contribution. We also have a [feature wishlist](https://github.com/tlaplus/tlaplus/blob/master/general/docs/contributions.md).
```

Comment:
This disclaimer should *not* be removed, because our resources are too constrained to welcome arbitrary contributions.

Reply:
I'll be doing some updates to CONTRIBUTING.md shortly

Reply:
I added a variant of the disclaimer.